builds/unix/ftconfig.h.in

```diff
@@ -0,0 +1,603 @@
+/***************************************************************************/
```

Comment:
How did you create this file?

Reply:
adapted copy of builds/unix/ftconfig.in , this was just too terrible to port:

https://git.savannah.gnu.org/cgit/freetype/freetype2.git/tree/CMakeLists.txt#n221

Reply:
Good question btw

Reply:
Please add a comment saying so. It will help a lot to anyone reading the wrap later.

Reply:
Added a comment indicating provenance and purpose of modifications